cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -650,16 +650,21 @@ share
     UniqueOffChainPoolFetchError poolId fetchTime retryCount
     deriving Show
 
-  OffChainAnchorData
-    votingAnchorId          VotingAnchorId      noreference
-    hash                    ByteString
-    json                    Text                sqltype=jsonb
-    bytes                   ByteString          sqltype=bytea
+  OffChainVoteData
+    votingAnchorId      VotingAnchorId      noreference
+    hash                ByteString
+    json                Text                sqltype=jsonb
+    bytes               ByteString          sqltype=bytea
+    UniqueOffChainVoteData votingAnchorId hash
+    deriving Show
 
-  OffChainAnchorFetchError
```

Comment:
These 2 tables that are deleted, should be deleted also from the sql files.

Reply:
having a bit of trouble with the rebasing!! just reran a migration but it's doing weird things it seems